src/lib/components/BucketAggregation/BucketAggregation.js

```diff
@@ -51,6 +51,14 @@ class BucketAggregation extends Component {
   _getResultBuckets = (resultsAggregations) => {
     // get buckets of this field
     const thisAggs = _get(resultsAggregations, this.agg.aggName, {});
+    if (thisAggs['buckets'] && !Array.isArray(thisAggs['buckets'])) {
```

Comment:
I would also a quick comment inside the if to explain: `// buckets can be objects or arrays: convert to array if object to keep it consistent`
I would also change the last `return` to avoid the double if:

```
if (...) {
    return thisAggs['buckets']
}
return [];